aspnetcore/test/troubleshoot.md

```diff
@@ -19,56 +19,59 @@ The following links provide troubleshooting guidance:
 
 ### Both the 32 bit and 64 bit versions of the .NET Core SDK are installed
-In the **New Project** dialog for ASP.NET Core, you may see the following warning: 
 
-    Both 32 and 64 bit versions of the .NET Core SDK are installed. Only templates from the 64 bit version(s) installed at C:\Program Files\dotnet\sdk\" will be displayed.
+In the **New Project** dialog for ASP.NET Core, you may see the following warning:
+
+> Both 32 and 64 bit versions of the .NET Core SDK are installed. Only templates from the 64 bit version(s) installed at 'C:\\Program Files\\dotnet\\sdk\\' will be displayed.
 
 ![A screenshot of the OneASP.NET dialog showing the warning message](troubleshoot/_static/both32and64bit.png)
 
-This warning appears when both 32-bit (x86) and 64-bit (x64) versions of the [.NET Core SDK](https://www.microsoft.com/net/download/all) are installed. Common reasons both versions can be installed include:
+This warning appears when both 32-bit (x86) and 64-bit (x64) versions of the [.NET Core SDK](https://www.microsoft.com/net/download/all) are installed. Common reasons both versions may be installed include:
 
-* You originally downloaded the .NET Core SDK installer using a 32-bit machine, but then copied it across and installed it on a 64-bit machine. 
+* You originally downloaded the .NET Core SDK installer using a 32-bit machine but then copied it across and installed it on a 64-bit machine.
 * The 32-bit .NET Core SDK was installed by another application.
 * The wrong version was downloaded and installed.
 
 Uninstall the 32-bit .NET Core SDK to prevent this warning. Uninstall from **Control Panel** > **Programs and Features** > **Uninstall or change a program**. If you understand why the warning occurs and its implications, you can ignore the warning.
 
 ### The .NET Core SDK is installed in multiple locations
-In the **New Project** dialog for ASP.NET Core you may see the following warning: 
 
- The .NET Core SDK is installed in multiple locations. Only templates from the SDK(s) installed at 'C:\Program Files\dotnet\sdk\' will be displayed.
+In the **New Project** dialog for ASP.NET Core, you may see the following warning:
+
+> The .NET Core SDK is installed in multiple locations. Only templates from the SDK(s) installed at 'C:\\Program Files\\dotnet\\sdk\\' will be displayed.
 
 ![A screenshot of the OneASP.NET dialog showing the warning message](troubleshoot/_static/multiplelocations.png)
 
-You see this message when you have at least one installation of the .NET Core SDK in a directory outside of *C:\Program Files\dotnet\sdk\*. Usually that happens when the .NET Core SDK has been deployed on a machine using copy/paste instead of the MSI installer.
+You see this message when you have at least one installation of the .NET Core SDK in a directory outside of *C:\\Program Files\\dotnet\\sdk\\*. Usually this happens when the .NET Core SDK has been deployed on a machine using copy/paste instead of the MSI installer.
```

Comment:
why not use forward slash? Folks know what you mean. `c:/Program Files/dotnet/sdk` works when you copy/paste and looks alot better in raw format.

@tdykstra  what do you think?

Reply:
I agree, but it was a Windows directive I think. I think the guidelines are when only talking about Windows to use the backslash.

Reply:
Yes, the guideline is ...

> To indicate a path, first type the drive name, followed by a colon and a backslash. Then type the name of each folder in the order that you would open it, separated by a backslash, as follows:
>
> C:\Documents and Settings\user1

Reply:
Do you want me to go ahead anyway with forward slashes?

Reply:
Where did that quotation come from?

Reply:
@guardrex  keep what you have, but I'd like to think about using my approach.

Reply:
MS Writing Style Guide - Section 5.18.8 path